src/stitch_fix/logWeasel.js

```diff
@@ -1,19 +1,12 @@
 import { ulid } from 'ulid';
 import { constantCase } from 'constant-case';
 
-let globalKey;
+const generateId = (key) => {
```

Comment:
This is a bit of a subjective/taste/branding thing, but one thing I liked about the singleton was that it had `LogWeasel` in the name: `LogWeasel.generateId()`. It's a bit more opaque now, since folks can `import generateId` and call `generateId(key)`. I might prefer, `generateLogWeaselId(key)` so it's clear what `id` is being generated and you don't have to look at the import. Thoughts?

Reply:
Yah, I hear you. The connection remains in the import statement: `import generateId from '@stitch-fix/log-weasel'`, and as a default export, people can import it with whatever name they like. If it makes their code more readable they could import as `import generateLogWeaselId from '@stitch-fix/log-weasel'`. 

I had the exact same reaction as the prevalence of importing single functions instead of helper objects was increasing, but it's pretty standard now. I would say we have both our bases covered -- the common way of importing now, plus an escape hatch if people find that `generateID` is too generic for their use case. WDYT?